### PR TITLE
Add RBD and DBD dates to submitted applications CSV

### PR DESCRIPTION
**Pull request by tvararu**
## Context

When recalculating dates, it's good to be able to compare these attributes in bulk.

## Changes proposed in this pull request

- Add `send_data` to controller to download as CSV file with embedded timestamp
- Add RBD and DBD dates to submitted applications CSV

## Guidance to review

:ship:

Screenshot 2020-04-25 at 12 37 20

## Link to Trello card

https://trello.com/c/qNxlrf6g/1406-add-rbd-dbd-data-to-the-submitted-application-choices-csv

## Things to check

- [x] This code doesn't rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] API release notes have been updated if necessary
- [x] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training#azure-hosting-devops-pipeline)

### CODE REVIEW

File: app/views/support_interface/performance/index.html.erb [hunk 1]
```diff
@@ -60,7 +60,7 @@
 </p>
 
 <p class='govuk-body'>
-  <%= govuk_link_to 'Download candidate survey results (CSV)', support_interface_candidate_survey_path %>
+  <%= govuk_link_to 'Download candidate survey results (CSV)', support_interface_candidate_survey_path, download: 'apply-candidate-survey-results.csv' %>
```
Comment (duncanjbrown) [inline app/views/support_interface/performance/index.html.erb:63]: If there are no survey results the else branch of `SupportInterface#candidate_survey` runs which redirects to a HTML 404 page and we get HTML in the CSV. Should we `head 404, content_type: 'text/csv'` or sth?

Screenshot 2020-04-27 at 09 25 15

Comment (tvararu) [reply app/views/support_interface/performance/index.html.erb:63]: Haha, Quicklook still trying to display that malformed CSV. I'll give that suggestion a go 👍

File: app/views/support_interface/performance/index.html.erb [hunk 2]
```diff
@@ -50,7 +50,7 @@
 </p>
 
 <p class='govuk-body'>
-  <%= govuk_link_to 'Download referee survey results (CSV)', support_interface_referee_survey_path %>
+  <%= govuk_link_to 'Download referee survey results (CSV)', support_interface_referee_survey_path, download: 'apply-referee-survey-results.csv' %>
```
Comment (davidgisbey) [inline app/views/support_interface/performance/index.html.erb:53]: The provider user and referee/candidate survey controller actions create a CSV as an attachment. I'm not 100% sure how this interacts with the HTML download attribute.

Comment (tvararu) [reply app/views/support_interface/performance/index.html.erb:53]: 👍 I didn't check, will remove

File: app/views/support_interface/performance/index.html.erb (hunk 1 above)
Comment (tvararu) [reply app/views/support_interface/performance/index.html.erb]: Removed it in the end on the links that don't pertain to this PR!

Comment (tvararu) [reply app/views/support_interface/performance/index.html.erb]: Ah nevermind still need to fix the 404 case... sorry

**Pull request merged**